clusterloader2/pkg/config/template_functions.go

```diff
@@ -55,14 +57,18 @@ func GetFuncs() template.FuncMap {
 		"RandInt":       randInt,
 		"RandIntRange":  randIntRange,
 		"Seq":           seq,
+		"SliceOfZeros":  sliceOfZeros,
 		"SubtractFloat": subtractFloat,
 		"SubtractInt":   subtractInt,
 		"YamlQuote":     yamlQuote,
 	}
 }
 
+// seq returns a slice of size 'size' filled with zeros.
+// Deprecated: Naming generates confusion. Please use 'SliceOfZeros' for explicit zero values or 'Loop' for.
```

Comment:
... or 'Loop' for. 

Isn't there missing anything?

Reply:
It was missing.